### PR TITLE
migrate to m105

### PR DESCRIPTION
**Pull request by SergeevPavel**

### CODE REVIEW

**Review by igordmn — APPROVED** [review]

File: skiko/build.gradle.kts [hunk 1]
```diff
@@ -145,10 +145,12 @@ fun compileNativeBridgesTask(os: OS, arch: Arch): TaskProvider<CompileSkikoCppTa
                 val iosArchFlags = when (arch) {
                     Arch.Arm64 -> arrayOf(
                         "-target", "arm64-apple-ios",
+                        "-mios-version-min=11.0",
```
Comment (igordmn) [inline skiko/build.gradle.kts:148]: Is it added because this is the minimal version supported by Skia?

Comment (SergeevPavel) [reply skiko/build.gradle.kts:148]: Yes, the new version of skia uses some language features that are allowed only from ios11

**Comment by SergeevPavel** [comment]
Looks like we are ready to merge. @igordmn what do you think? Also can we issue a new version of skiko after that?

**Comment by igordmn** [comment]
Let's merge. Github Actions seems hangs, but the previous run was almost successful, except wasm still weren't published.

**Comment by igordmn** [comment]
0.7.28 is building

**Pull request merged**